src/RecoveryServices/RecoveryServices/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 13.0.0
```

Comment:
DO NOT add a new version. Please add the change log under `Upcoming Release` section.

Reply:
@vidai-msft Fixed